src/komponenter/din-situasjon/situasjon.test.tsx

```diff
@@ -0,0 +1,116 @@
+import * as React from 'react';
+import { render, screen } from '@testing-library/react';
+import Situasjon from './situasjon';
+import { ComponentType, ReactChildren, ReactElement } from 'react';
```

Comment:
Noen grunn til å hente i dette når du allerede har hentet inn hele React?
Eller er det forberedelser til 17 når vi kan kutte topp-importen?

Reply:
Dette er kun et resultat av at IntelliJ automatisk legger inn riktige imports for meg. :-) Jeg kunne ha brukt 
`React.ComponentType` i stedet, for å fungere med `import * as React …`, men jeg synes egentlig tanken om forberedelser til 17 var god, så jeg tror jeg lener meg på den.